src/types/constants.go

```diff
@@ -41,4 +41,10 @@ const (
 	// K8sNetworkPolicy
 	K8sNwPolicyAPIVersion = "networking.k8s.io/v1"
 	K8sNwPolicyKind       = "NetworkPolicy"
+
+	// Operation Types
+	SumOpTypeFile    = "File"
+	SumOpTypeProcess = "Process"
+	SumOpTypeNetwork = "Network"
+	SumOpTypeSyscall = "Syscall"
```

Comment:
Aren't these constants already defined? Maybe we need to just add `Syscall` but rest are already there.

Reply:
Yes. Constants exists in [systempolicy](https://github.com/accuknox/discovery-engine/blob/ce809848f042feb4bd95f8afa7abd5f3e680ed4b/src/systempolicy/systemPolicy.go#L52) 

Should I import systempolicy and use those constants or leave it as it is?

Reply:
Couldn't import systempolicy into obserability because it has "import cycle error".  We may introduce types interface and break the cycle. I guess it's not a good approach. 
If you have any thoughts on this, please let me know.

Reply:
Import error cycle error has found in systempolicy package

```
import cycle not allowed: import stack: [command-line-arguments github.com/accuknox/auto-policy-discovery/src/systempolicy github.com/accuknox/auto-policy-discovery/src/feedconsumer github.com/accuknox/auto-policy-discovery/src/plugin github.com/accuknox/auto-policy-discovery/src/observability github.com/accuknox/auto-policy-discovery/src/systempolicy]
```
Since `systempolicy` package using `plugin` package and the `plugin` has `observability`. So we couldn't `systempolicy` in `observability`.
cc: @Ankurk99

Reply:
I replaced the constants in `systempolicy` package with constants from `types/constants.go`. So for now we have system operation constants in one place.